fast/stages/02-networking-nva/vpc-landing.tf

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# tfdoc:file:description Landing VPC and related resources.
+
+module "landing-project" {
+  source          = "../../../modules/project"
+  billing_account = var.billing_account_id
+  name            = "prod-net-landing-3" #TODO - reset counter
+  parent          = var.folder_id
+  prefix          = var.prefix
+  service_config = {
+    disable_on_destroy         = false
+    disable_dependent_services = false
+  }
+  services = [
+    "compute.googleapis.com",
+    "dns.googleapis.com",
+    "iap.googleapis.com",
+    "networkmanagement.googleapis.com",
+    "stackdriver.googleapis.com"
+  ]
+  shared_vpc_host_config = {
+    enabled          = true
+    service_projects = []
+  }
+}
+
+# Untrusted VPC
+
+module "landing-untrusted-vpc" {
+  source     = "../../../modules/net-vpc"
+  project_id = module.landing-project.project_id
+  name       = "prod-untrusted-landing-0"
+  mtu        = 1500
+
+  dns_policy = {
+    inbound  = true
+    logging  = false
+    outbound = null
+  }
+
+  # Set explicit routes for googleapis in case the default route is deleted
+  routes = {
+    private-googleapis = {
+      dest_range    = "199.36.153.8/30"
+      priority      = 1000
+      tags          = []
+      next_hop_type = "gateway"
+      next_hop      = "default-internet-gateway" #TODO: route to ILB 
```

Comment:
Should untrusted even have routes for private/restricted?

Reply:
You're right. Let's remove them